example/qcloud_pipe_merged.mzqc

```diff
@@ -0,0 +1,130 @@
+{ "mzQC": {
+  "creationDate": "2019-10-30T12:20:26",
+  "version": "0_0_11",
+  "runQualities": [
+    {
+      "metadata": {
+        "inputFiles": [
+          {
+            "location": "file:///dev/null",
+            "name": "qcloud_pipe_merged.json",
+            "fileFormat": {
+              "cvRef": "MS",
+              "accession": "MS:1000584",
+              "name": "mzML format"
+            },
+            "fileProperties": [
+              {
+                "cvRef": "MS",
+                "accession": "MS:1000747",
+                "name": "completion time",
+                "value": "2019-10-30 15:26:50.093306"
+              },
+              {
+                "cvRef": "MS",
+                "accession": "MS:1000569",
+                "name": "SHA-1",
+                "value": "3f98581e6291b298c2a11a4410d7198e"
```

Comment:
BTW, which type of checksum is it you want the zip file name to contain? We'd need that for any independent verification.